packages/docs/src/locales/en/en.json

```diff
@@ -2967,71 +2967,71 @@
   },
   "overview": {
     "title": "What is Vuestic UI",
-    "description": "We’ve designed the framework in such a way that it satisfies the most sophisticated design fantasies and allows you to implement them",
+    "description": "We’ve designed the framework in such a way that it satisfies the most sophisticated design fantasies and allows you to implement them.",
     "featuresOverview": "Features overview",
     "vueThreeCompatible": "Fully **Vue.js 3 compatible**",
-    "featureRich": "Contains a **50+ feature rich components** that can be used in any design solution",
-    "configurable": "Allows **to configure components** components globally via config and css variables",
+    "featureRich": "Contains a **50+ feature-rich components** that can be used in any design solution",
+    "configurable": "Allows **to configure components** globally via config and CSS variables",
     "responsive": "Fully **responsive** and **supports modern browsers** (no IE11, sorry)",
     "translatable": "Integrates seamlessly with **i18n**"
   },
   "translation": {
     "title": "Translation",
-    "description": "Translation is only needed for documentation (`packages/docs/locales`) as components don't use any translation strings at the moment. After finishing work on the translation, do not forget to change its status to `full` in file `locales/index.ts`. Here's the current state for languages support:",
+    "description": "For the moment being we only provide translations for the main documentation pages (such as Contribution, Getting Started, etc.). The actual translation-strings could be found at `packages/docs/locales`. Currently there are no translations available for the components' docs. After you finish working on a translation for some page, please, do not forget to update its status to `full` in the `locales/index.ts` file. Here's the current languages-support state:",
     "table": {
       "language": "Language",
       "code": "Code",
-      "supported": "Supported"
+      "supported": "Support"
     },
     "language": {
       "en": "English",
       "ru": "Russian",
-      "es": "Spain"
+      "es": "Spanish"
     },
     "status": {
-      "part": "Partially",
-      "full": "Fully"
+      "part": "Partial support",
+      "full": "Full support"
     }
   },
   "table": {
     "title": "Tables",
-    "summaryText": "You can use the class `.va-table` for customizing styles of your tables. Here is the couple examples",
+    "summaryText": "Feature-rich Data-Tables are currently a work in progress. However, for now you can use plain `<table>`s with the `.va-table` class that would apply Vuestic-theme styles to them. Here is a couple of examples.",
```

Comment:
"are currently under development"

Reply:
Nice wording. Exactly what I was looking for.